buildkit/topn.yaml

```diff
@@ -8,7 +8,8 @@ description: TopN is an open source PostgreSQL extension that returns the top va
 licence: AGPL-3.0
 keywords:
   - top values
-  - ranking events
+  - ranking
+  - events
```

Comment:
I would put this back to `ranking events` because each of the word can't describe what `topn` is unless combined.